docs/modules/maintain-cluster/pages/rolling-upgrades.adoc

```diff
@@ -5,6 +5,18 @@
 
 {description}
 
+== Best Practices
+
+Before starting a rolling upgrade, consider the following best practises:
```

Comment:
```suggestion
Before starting a rolling upgrade, consider the following best practices:
```

Reply:
Dagnamit! When will those US English practitioners learn how to use words properly ;-)